package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "eslint src"
+    "lint": "eslint src",//"**/*.js "check every js files
```

Comment:
Do not leave unnecessary comments in your files. In a JSON formatted file comments does not exist so basically this file is invalid and won't work. Use the `**/*.js` version to check every JavaScript file